src/plugins/intel_gna/tests/unit/transformations/gna_pwl.cpp

```diff
@@ -9,19 +9,34 @@
 #include <transformations/init_node_info.hpp>
 #include <type_traits>
 
+#include "common/numerical_utils.hpp"
 #include "common_test_utils/data_utils.hpp"
 #include "common_test_utils/ngraph_test_utils.hpp"
+#include "ngraph_functions/builders.hpp"
+#include "openvino/opsets/opset11.hpp"
+#include "openvino/runtime/intel_gna/properties.hpp"
 #include "transformations/pwl_approximation.hpp"
 
+using namespace ov::intel_gna;
 using namespace ov::intel_gna::common;
 
+constexpr double kAccuracyMaxErrorPrecentage = 0.09;
+constexpr double kPerformanceMaxErrorPrecentage = 1.0;
+
+double get_max_error_percentage(const PWLApproximationMode& mode) {
```

Comment:
in fact we don't need const& here, passing by value might spare us 4B? + a bit more elegant?

Reply:
yes I agree, but in many places in the code we use that convention, this is why I wanted to align, may change

Reply:
.... then, there is still ConfigurationSigmoid::create_config(PWLApproximationMode mode) left then.
choose whatever way better for you then.